hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -210,6 +210,14 @@ public void close() throws IOException {
     flush();
     output.close();
     output = null;
+    Path path = logFile.getPath();
```

Comment:
@hddong : Any possible reasons why blank file is created in the first place ?

Reply:
> @hddong : Any possible reasons why blank file is created in the first place ?

Create blank file for `appendBlock` when new `HoodieLogFormatWriter`
https://github.com/apache/incubator-hudi/blob/f1592be629c3f9762f62d4e1dbf3be54f213d92d/hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java#L105-L108
But there is a special case,  when roll over is true(block size is past the threshold), we will close the old writer and create a new writer . And if we close a new writer created by `rolloverIfNeeded` , there will left a blank file.

Reply:
fair enough. but this is effectively doing an extra RPC every close, just to handle this corner case. what is the actual  issue caused by the empty file? does the query/writing fail or get stuck?